docs/tutorials/simple-date-clock-application.adoc

```diff
@@ -0,0 +1,435 @@
+
+Cloud harness allows you to quickly setup app that needs to be ran in a kubernete cluster.
+It generates the initial files and folders for your project depending on some templates tacking different aspects of your app depending on your requirements, __e.g.__, for a webapp project, it generates the frontend initial files for ReactJS and the initial Flask files for the backend.
+For the API part, cloud harness relies on OpenAPI 3 to deal with the endpoints/model description.
+
+The different aspects that will be covered here are:
+
+* how to install cloud-harness;
+* how to quickly setup a first version of a local cluster using minikube;
+* how to build and locally deploy the projects from the repository;
+* how to bootstrap a new app, build it and run it;
+* how to modify/update the app, built it and run it again.
+
+== The tools you need
+
+Before installing everything, please be sure you have the following tools installed on your machine:
+
+* `python`
+* `minikube` -- the local mini cluster
+* `kubectl` -- to help controlling the cluster
+* `helm` -- to deal with Kubernete "packages" (helm is basically a package manager for Kubernete)
+* `skaffold` -- to build/deploy/run the apps
+
+
+== Install cloud-harness (if it is not yet done)
+
+The installation is pretty straight forward.
+Cloud-harness is coded in Python, consequently, it's always better to create a local virtualenv dedicated to the project to avoid messing with your system dependencies.
+As shown in the snippet above, we will clone the repository, change directory inside the freshly clone directory and create a virtualenv inside (for this tutorial, the Python's version used is CPython 3.10.6).
+
+.Cloning and installing cloud-harness
+[source,bash]
+----
+git clone git@github.com:MetaCell/cloud-harness.git
+cd cloud-harness
+python -m venv --symlinks venv     # this creates a virtualenv in the "venv" directory
+----
+
+Now that the virtualenv is created, we need to activate it to work in isolation.
+
+.Activating the virtualenv and installing cloud-harness
+[source,bash]
+----
+source venv/bin/activate
+sh ./install.sh
+----
+
+To test if your installation went successfully, check if at least the following command exist: `harness-application`.
+
+.Checking the installation
+[source,bash]
+----
+harness-application --help
+----
+
+If everything is good, you're in the right path to use cloud-harness!
+
+
+== How to setup your local cluster using minikube (if not done yet)
+
+Cloud-harness is designed to help you to generate all the required artifacts for a deployment on a Kebernete cluster.
+For a local deployment, you can use minikube, which is basically https://minikube.sigs.k8s.io/docs/start/[a small Kubernete cluster on your machine].
+
+First, we will create a minikube cluster with 6Gb of memory, 4 CPUs and a disk of more or less 2GB.
```

Comment:
these requirements are overkill for a simple application, CH has no requirements really

Reply:
You're right, it's more related to the deployent than the application in itself. I'll be more specific and detail better what CH needs and what's related to the deployment in itself.